cmd/run.go

```diff
@@ -62,6 +63,7 @@ func buildRunCommand() *cobra.Command {
 	runCmd.PersistentFlags().BoolVar(&ttpCfg.NoCleanup, "no-cleanup", false, "Disable cleanup (useful for debugging and daisy-chaining TTPs)")
 	runCmd.PersistentFlags().UintVar(&ttpCfg.CleanupDelaySeconds, "cleanup-delay-seconds", 0, "Wait this long after TTP execution before starting cleanup")
 	runCmd.Flags().StringArrayVarP(&argsList, "arg", "a", []string{}, "variable input mapping for args to be used in place of inputs defined in each ttp file")
+	runCmd.Flags().StringArrayVarP(&targetsList, "target", "t", []string{}, "variable input mapping to specify targets")
```

Comment:
In any case, I don't think this "you can pass targets on the CLI" use case would make sense. It seems to be blurring three different notions together: 

1. Where a TTP is capable of being run (ex: can't run macOS TTP on windows) 
2. Where you are currently running a TTP
3. Command arguments for control flow 

Regarding (1) I think what you really want to do here is just have a pre-execution check that you are running on a compatible platform, so that the user receives an intelligible error as early as possible in the TTP execution process. 

Regarding (2): For instance, if a TTP supports both macOS and linux, and you need to check which one you are executing on at runtime in order to modify TTP behavior (for instance, using a different variant of the `find` command), then you should just reference `runtime.GOOS` (possibly packed into ExecutionConfig in some form) - a user passing a different target on the command line doesn't actually change the platform on which you are running. 

Regarding (3): A risk with the current approach is that you end up re-implementing everything that `args` handles as far as validation goes. That'll be tricky to centralize - will need to be careful